lib/creator/transformations/init-transforms.js

```diff
@@ -0,0 +1,39 @@
+const j = require('jscodeshift');
+
+module.exports = function (promptOptions) {
+	const promptParams = Object.keys(promptOptions);
+	if (promptParams && promptParams.length) {
+		let ast = j('{}');
```

Comment:
That's not the `Object` as you might expect. This is a block statement. See https://astexplorer.net/#/gist/8ae05b2650bc27d2c4923ef8e6f36241/0a32afa94ee7cd5ecd742958c815706d34201345